packages/explat-client/src/create-explat-react-client.ts

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
```

Comment:
This file is just me implementing the `useExperiment` (or `useExperimentAssignment` as I may call it) hook.

It is a demo of how` getExperimentAssignment` can be used.